.Gruntfile.babel.js

```diff
@@ -227,7 +234,7 @@ function browserifyCfg ({straight, lazy}, cfg = {}) {
   function gencfg (out, paths, isLazy) {
     var c = {
       options: {
-        transform: [script2ify, 'vueify', ejsify, 'babelify'],
+        transform: [localify, script2ify, 'vueify', ejsify, 'babelify'],
```

Comment:
I would not have personally chosen to do this in grunt on every build... but I guess we can change that later if it impacts build times.